src/Features/Core/Portable/ConvertAnonymousType/AbstractConvertAnonymousTypeToClassCodeRefactoringProvider.cs

```diff
@@ -122,21 +121,18 @@ private async Task<Document> ConvertAsync(Document document, TextSpan span, bool
                 containingMember, anonymousObject,
                 anonymousType, cancellationToken).ConfigureAwait(false);
 
+            var context = new CodeGenerationContext(
+                generateMembers: true,
+                sortMembers: false,
+                autoInsertionLocation: false);
+
+            var codeGenOptions = await CodeGenerationOptions.FromDocumentAsync(context, document, cancellationToken).ConfigureAwait(false);
```

Comment:
nit: i wish 'document' was the first parameter here.  esp since it's called FromDocument.